internal/config/config.go

```diff
@@ -12,8 +12,9 @@ import (
 )
 
 const (
-	JoyrcFile     = ".joyrc"
-	JoyDefaultDir = ".joy"
+	CatalogConfigFile = "joy.yaml"
```

Comment:
I think we need to be delicate here since the catalog repo now controls the joy-generator, and if we don't update the generator to a compatible version this change could break things with argocd.

I would update this such that the catalogConfigFile is a list of candidates: .joyrc and joy.yaml, such that it looks for joy.yaml first and falls back to .joyrc.

Then we can update the joy-generator, then we can update the catalog.

Reply:
I have merged https://github.com/nestoca/catalog/pull/597 yesterday and joy/joy generator have been working the same.

Reply:
@davidmdm the symlink does the backward compatibility magic here.